extensions/wikia/Wall/Wall.i18n.php

```diff
@@ -128,6 +128,8 @@
 	'wall-action-move-thread-ok' => 'Move thread',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "wall-action-move-thread-ok" 28 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)